test/factorization/cholesky_kernels.cpp

```diff
@@ -46,14 +48,17 @@ class CholeskySymbolic : public CommonTestFixture {
         matrices.emplace_back(
             "example small",
             gko::initialize<matrix_type>(
-                {{1, 0, 1, 0}, {0, 1, 0, 1}, {1, 0, 1, 0}, {0, 0, 0, 1}}, ref));
+                {{1, 0, 1, 0}, {0, 1, 0, 1}, {1, 0, 1, 0}, {0, 1, 0, 1}}, ref));
+        // this is the example from Liu 1990 https://doi.org/10.1137/0611010.
+        // "The Role of Elimination Trees in Sparse Factorization"
```

Comment:
I do not have the access to the paper unfortunately. Which one is the example from paper, example small or example?